docs/tutorials/pre_executed/nestedframe.ipynb

```diff
@@ -0,0 +1,2041 @@
+{
```

Comment:
<p>Maybe ".., pandas functions that we have augmented for nested functionality are modified to allow ... ". Otherwise, it looks like a typo unless you read it out loud to yourself.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Point taken, I felt similarly when doing another pass. I started with your suggestion but ended up with a different sentence here, let me know if it's still an issue